third_party/angular-material/README.md

```diff
@@ -0,0 +1,22 @@
+# AngularJS
+
+## URL
+
+https://github.com/angular/material/releases/tag/v1.1.4
+
+## Version
+
+1.1.4
+
+## License
+
+MIT License
+
+## License File
+
+[LICENSE](repo/LICENSE)
+
+## Description
+Material Design is a specification for a unified system of visual, motion, and interaction design that adapts across different devices. Our goal is to deliver a lean, lightweight set of AngularJS-native UI elements that implement the material design specification for use in AngularJS single-page applications (SPAs).
+
+This repo only includes the minify version files and LICENSE file.
```

Comment:
Change to: "Note: only the minimum set of files necessary to support the OpenThread project are included here."